### PR TITLE
Upgrade to NoFlo 0.7.x

### PR DESCRIPTION
**Pull request by bergie**
Needed for #68

### CODE REVIEW

**Comment by bergie** [comment]
@jonnor green now

File: package.json [hunk 1]
```diff
@@ -48,8 +47,8 @@
       "TestDelayOrError": "components/TestDelayOrError.coffee"
     },
     "graphs": {
-      "RepeatTest": "spec/fixtures/repeat.fbp",
-      "IIPKickTest": "spec/fixtures/kickiip.fbp"
+      "RepeatTest": "graphs/repeat.fbp",
```
Comment (jonnor) [inline package.json]: Why is this? Fine to have a convention, but to have hardcoded expectations is not cool - especially when it used to not be.

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ language: node_js
 services:
 - rabbitmq
 node_js:
-- '0.10'
+- '4.2'
```
Comment (jonnor) [inline .travis.yml]: This breaks autodeploy due to NPM version/keyformat change, so you will need to update that as well.

File: graphs/kickiip.fbp [hunk 3]
```diff
@@ -1,4 +1,5 @@
-# !runtime noflo
+# @runtime noflo-nodejs
+# @name IIPKickTest
```
Comment (jonnor) [inline graphs/kickiip.fbp]: Is this mandatory now too? Since when?

File: package.json (hunk 1 above)
Comment (bergie) [inline package.json]: Yes, this is because we're now using fbp-manifest instead of list-installed.

File: graphs/kickiip.fbp (hunk 3 above)
Comment (bergie) [inline graphs/kickiip.fbp]: If the library had been using grunt-noflo-manifest, these would've been needed already long time ago.

Comment (bergie) [inline graphs/kickiip.fbp]: Note: only needed when filename doesn't conform to the desired component name (as is often case with components where we need a separate implementation for noflo-nodejs and noflo-browser)

File: .travis.yml (hunk 2 above)
Comment (bergie) [inline .travis.yml]: Done

File: package.json (hunk 1 above)
Comment (jonnor) [inline package.json]: Why can't one have graphs whereever one would like with fbp-manifest?
This breaking change is also not mentioned in https://github.com/noflo/noflo/blob/master/CHANGES.md

File: graphs/kickiip.fbp (hunk 3 above)
Comment (jonnor) [inline graphs/kickiip.fbp]: If the name comes from @name and things can't be in non-standard paths (see previous comment), then whats the point of having data in `package.json` at all?

Comment (bergie) [inline graphs/kickiip.fbp]: No point whatsoever in regards to newer NoFlo, only there for usage with old NoFlo.

Comment (jonnor) [inline graphs/kickiip.fbp]: @bergie can you please document these things in CHANGES.md? otherwise how are people to know...

Comment (bergie) [inline graphs/kickiip.fbp]: https://github.com/noflo/noflo/commit/ad8bfbd4f1d8ebeeac90438d6401a577d681629f

**Pull request merged**